traits/ctraits.c

```diff
@@ -4529,28 +4546,18 @@ static trait_setattr setattr_property_handlers[] = {
     (trait_setattr)post_setattr_trait_python, NULL};
 
 static PyObject *
-_trait_property(trait_object *trait, PyObject *args)
+_set_property(trait_object *trait, PyObject *args)
```

Comment:
Please rename this C function to `_trait_set_property` (and similarly for `_get_property`), for consistency with the names of the C functions backing all the other `cTrait` methods. The Python-visible names should remain `_set_property` and `_get_property`, of course.

Reply:
Done.